src/assets/news.md

```diff
@@ -0,0 +1,3 @@
+# This is the news
+
+*Some* testing **here**
```

Comment:
@Nutomic I looked through all our announcements, and couldn't find anything to put here that isn't one of our releases.

Reply:
Some examples:

- federation first being enabled on lemmy.ml
- [lemmy.ml switching to open federation](https://lemmy.ml/post/79716)
- [promoting lemmy](https://lemmy.ml/post/76774)
- the history of lemmy (on the about page) could also be moved to a blog post, and linked from the frontpage
- the thing i'm writing about free instance hosting should also go here

Reply:
I can add a few of these.